doc/source/whatsnew/v0.24.2.rst

```diff
@@ -31,6 +31,7 @@ Fixed Regressions
 - Fixed regression in ``IntervalDtype`` construction where passing an incorrect string with 'Interval' as a prefix could result in a ``RecursionError``. (:issue:`25338`)
 - Fixed regression in :class:`Categorical`, where constructing it from a categorical ``Series`` and an explicit ``categories=`` that differed from that in the ``Series`` created an invalid object which could trigger segfaults. (:issue:`25318`)
 - Fixed pip installing from source into an environment without NumPy (:issue:`25193`)
+- Fixed bug that :class:`ExtensionArray` cannot be used to matplotlib plotting (:issue:`25587`)
```

Comment:
"Fixed bug where :class:`ExtensionArray` could not be used in matplotlib plotting (:issue:`25587`)"

Reply:
ExtensionArray isn't in the top-level API so it would have to be something like :class:\`api.extensions.ExtensionArray\`

Reply:
Fixed.